aptos-move/aptos-vm/src/move_vm_ext/session/mod.rs

```diff
@@ -288,6 +297,11 @@ where
         txn_context.mark_unbiasable();
     }
 
+    pub(crate) fn mark_system_entry_function(&mut self) {
```

Comment:
I would say that passing `&ModuleId` is safer here, so that we do not need to trust the caller to check for special address?